### DISCUSSION TITLE
Matches found after scan

### DISCUSSION THREAD

**Discussion by tusizi2016** [Divested-Mobile/hypatia#58] [Q&A]
What should i do?plz.

**Comment by SkewedZeppelin** [comment]
The detection are shown in your notifications area.

If you disabled notifications for the app, you need to grant the permission and scan again.

From the notifications you can chose to lookup a file on VirusTotal, delete the file, or ignore the detection.

**Reply by askusWT** [reply]
Do you want example false positives reported? For example I constantly pick up Whatsapp artifacts from interrupted voice recordings that virus total shows are not risky.

**Reply by SkewedZeppelin** [reply]
They are welcomed, but please note it is possible for some matches to be caused by the inherent false positive rate of bloom filters.
